pdm_build.py

```diff
@@ -10,7 +10,16 @@
     # https://github.com/astral-sh/ruff/issues/9528
     from subprocess import run  # noqa: S404
 else:
-    from nodejs.npm import run
+    from nodejs_wheel import executable
+
+    # Remove when https://github.com/njzjz/nodejs-wheel/pull/24 is merged
+    def run(cmd: List[str], check=True):
+        old_sys_argv = sys.argv
+        sys.argv = [old_sys_argv[0]] + cmd
+        try:
+            executable.npm()
+        finally:
+            sys.argv = old_sys_argv
```

Comment:
that `TYPE_CHECKING` block above was only needed because of an issue in nodejs-bin: https://github.com/samwillis/nodejs-pypi/pull/23

it can be removed since your `run` function is properly typed